README.md

```diff
@@ -1,19 +1,41 @@
-# React render() Must Not Have Side Effects
+# React render() must not have side effects
 
-## Objectives
+???
```

Comment:
The `???` is to indicate that this is a quiz - it's part of the syntax that Flatiron has cooked up to parse this .md file into questions.